hack/istio/install-bookinfo-demo.sh

```diff
@@ -200,6 +200,15 @@ if [ "${DELETE_BOOKINFO}" == "true" ]; then
   else
     $CLIENT_EXE delete namespace ${NAMESPACE}
   fi
+  # It takes some time to delete the namespace, let's wait for that
+  echo -n "Waiting for ${NAMESPACE} to be deleted..."
+  while $CLIENT_EXE get namespace ${NAMESPACE} &> /dev/null
+  do
+    echo -n '.'
+    sleep 10
+  done
+  echo ""
+
```

Comment:
```suggestion
    $CLIENT_EXE delete namespace ${NAMESPACE} --wait=true
  fi
```

Reply:
wait should be `true` by default. Does `oc project delete` not wait for the namespace to be deleted before returning?

Reply:
```
$ oc delete --help | grep wait=true
      --wait=true: If true, wait for resources to be gone before returning. This waits for finalizers.
```

Reply:
But I don't know if that is the default or not.

Reply:
Whatever is after the `=` is the default so `true` is the default. Does the `project` resource stick around until the `namespace` gets deleted or does `project` get removed without waiting for the `namespace` to be deleted?

Reply:
Either way adding `$CLIENT_EXE delete namespace` should fix it but I think you also need to add it to the `IS_OPENSHIFT=true` case.

Reply:
Added new commit using 'delete namespace'